src/views/AgmLocationChg.vue

```diff
@@ -441,12 +408,11 @@ export default class AgmLocationChg extends Mixins(CommonMixin, DateMixin,
     }
 
     // this is the id of THIS filing
-    // if 0, this is a new filing
-    // otherwise it's a draft filing
+    // it must be 0 (meaning new filing) -- we do not support resuming a draft filing
     this.filingId = +this.$route.params.filingId // number or NaN
 
     // if required data isn't set, go back to dashboard
-    if (!this.getIdentifier || isNaN(this.filingId)) {
+    if (!this.getIdentifier || this.filingId !== 0) {
```

Comment:
Checking the filing ID, which we pass as 0 in the router push call, ensures that the user cannot go to the URL manually.

```
this.$router.push({ name: Routes.AGM_LOCATION_CHANGE, params: { filingId: '0' } })
```